src/components/launch_alert_banner/launch_alert_banner.tsx

```diff
@@ -54,63 +58,83 @@ export function LaunchAlertBanner({
-              </Button>
-            </Feature>
-          </div>
-        ) : (
-          undefined
-        )
-      }
-    >
-      <div>
-        <LaunchMessage engagement={engagement} />
-        {engagement?.launch ? (
-          undefined
-        ) : (
-          <div>
+    <>
+      <DeleteModal
+        isOpen={activeModalKey?.includes(DELETE_ENGAGEMENT_MODAL_KEY)}
+        engagement={engagement}
+      />
+      <Alert
+        isInline
+        title={
+          engagement?.launch ? 'Engagement Launched' : 'Engagement Not Launched'
+        }
+        variant={statusAlert(overallStatus)}
+        actionLinks={
+          !engagement?.launch ? (
+            <div>
+              <Feature name={'writer'}>
+                <Flex>
+                  <FlexItem span={1}>
+                    <Button
+                      isDisabled={!isLaunchable}
+                      onClick={() => onLaunch(engagement)}
+                      data-cy={'launch_button'}
+                    >
+                      Launch
+                    </Button>
+                  </FlexItem>
+                  <FlexItem span={1}>
+                    <Button
+                      variant="danger"
+                      onClick={() => requestOpen(DELETE_ENGAGEMENT_MODAL_KEY)}
+                    >
+                      Delete
+                    </Button>
+                  </FlexItem>
+                </Flex>
+              </Feature>
+            </div>
+          ) : (
+            undefined
+          )
+        }
+      >
+        <div>
+          <LaunchMessage engagement={engagement} />
+          {!engagement?.launch ? (
             <div>
-              <span style={{ fontStyle: 'italic' }}>
-                {missingRequiredFields?.length !== 0
-                  ? 'Required fields are missing in the following sections:'
-                  : null}
-              </span>
-              <ul>
-                {Array.from(
-                  new Set(
-                    missingRequiredFields.map(
-                      field => ENGAGEMENT_FIELD_MAP[field]
+              <div>
+                <span style={{ fontStyle: 'italic' }}>
+                  {missingRequiredFields?.length !== 0
+                    ? 'Required fields are missing in the following sections:'
+                    : null}
+                </span>
+                <ul>
+                  {Array.from(
+                    new Set(
+                      missingRequiredFields.map(
+                        field => ENGAGEMENT_FIELD_MAP[field]
+                      )
                     )
-                  )
-                ).map((section: string, i) => (
-                  <li key={i}>
-                    <HashLink smooth to={`${path}#${section}`}>
-                      {ENGAGEMENT_CARDS[section]}
-                    </HashLink>
-                  </li>
-                ))}
-              </ul>
+                  ).map(section => (
+                    <li>
```

Comment:
add the key back to this `li`, or else we will get warnings in the browser console